src/main/java/by/exadel/internship/service/impl/FormServiceImpl.java

```diff
@@ -147,6 +167,47 @@ private void uploadFile(MultipartFile multipartFile, UUID id) {
 
     }
 
+    @Override
+    public void updateFeedback(UUID formId, FeedbackRequest feedbackRequest) {
+        MDCLog.putClassNameInMDC(SIMPLE_CLASS_NAME);
+        log.info("Try to save feedback for Form with uuid = {}", formId);
+        Form form = formRepository.findByIdAndDeletedFalse(formId).
+                orElseThrow(() -> new NotFoundException("Form with uuid = " + formId +
+                        " Not Found in DB", "form.uuid.invalid"));
+        UserDTO userDTO = userService.getById(feedbackRequest.getUserId());
+        setFeedbackByUserRole(userDTO,form,feedbackRequest);
+        log.info("Feedback was updating");
+
+    }
+
+    private void setFeedbackByUserRole(UserDTO userDTO, Form form, FeedbackRequest feedbackRequest){
+        log.info("Set feedback in Interview By userRole");
+        if (userDTO.getUserRole().equals(UserRole.ADMIN)){
```

Comment:
In case userDTO.getUserRole() = SUPER_ADMIN what expected behaviour should be? ignore or throw exception?

Reply:
done, I did throw exception